### PR TITLE
GP Internal Core API v1.1 : update TEE_AllocateOperation DSA

### PR DESCRIPTION
**Pull request by cedric-chaumont-st-dev**
add TEE_ALG_DSA_SHA224/TEE_ALG_DSA_SHA256

Signed-off-by: Cedric Chaumont <cedric.chaumont@st.com>

### CODE REVIEW

File: lib/libutee/tee_api_operations.c [hunk 1]
```diff
@@ -107,6 +107,8 @@ TEE_Result TEE_AllocateOperation(TEE_OperationHandle *operation,
 	case TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA384:
 	case TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA512:
 	case TEE_ALG_DSA_SHA1:
+	case TEE_ALG_DSA_SHA224:
+	case TEE_ALG_DSA_SHA256:
```
Comment (pascal-brand-st-dev) [inline lib/libutee/tee_api_operations.c]: Should we check the key size here?

**Comment by cedric-chaumont-st-dev** [comment]
Update

**Comment by cedric-chaumont-st-dev** [comment]
Rebase.

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by cedric-chaumont-st-dev** [comment]
Squahsed. Rebased.Tested.

**Pull request merged**